src/Tickets/Blocks/Ticket/app/editor/container-content/type/template.js

```diff
@@ -63,19 +54,21 @@ const Type = ({
 
 			<div className="tribe-editor__ticket__type-description">
 				<div>
-					{ TYPES[ type ]?.icon }
-					<span>{ __( TYPES[ type ]?.title, 'event-tickets' ) }</span>
+					{ ticketType.typeIcon }
+					<span>{ __( ticketType.typeName, 'event-tickets' ) }</span>
```

Comment:
Localization calls cannot be dynamic: they cannot happen on variables.
POT generation scripts are not executing the code, just parsing it: the `ticketType.typeName` variable will not be resolved and will either be provided to the translators incorrectly or not provided at all.
This is true for JS or PHP code.
Each string that should be localized must the provided in a static manner.
I left another note elsewhere.